addons/api/mirage/serializers/worker.js

```diff
@@ -7,4 +7,21 @@ import ApplicationSerializer from './application';
 
 export default ApplicationSerializer.extend({
   modelName: 'worker',
+
+  _hashForModel(model) {
+    const json = ApplicationSerializer.prototype._hashForModel.apply(
+      this,
+      arguments,
+    );
+    if (json.api_tags) {
+      const keys = Object.keys(model.api_tags);
```

Comment:
I get an error here in mirage:

<img width="513" alt="image" src="https://github.com/user-attachments/assets/31f91c39-8fea-46fb-838b-a240073dc4fe">

Reply:
What exactly triggered this error?

Reply:
trying to save an api_tag when there are no api_tags